src/plugins/chat/public/services/chat_service.ts

```diff
@@ -55,20 +55,30 @@ export class ChatService {
   // ChatWindow ref for delegating sendMessage calls to proper timeline management
   private chatWindowRef: React.RefObject<ChatWindowInstance> | null = null;
 
+  private get threadId() {
+    return this.threadId$.getValue();
+  }
+
   constructor() {
```

Comment:
what's the use case for plugins to get current chat thread id? this sounds like a internal stuff inside the chat.